virtualenvwrapper/workon.zsh

```diff
@@ -1,5 +1,6 @@
 export WORKON_HOME=$HOME/.virtualenvs
 export PROJECT_HOME=$HOME/Devel
+export VIRTUALENVWRAPPER_PYTHON=/usr/local/bin/python3
```

Comment:
This is expected to break across the multitude of different install locations for python3 on various platforms. When this breaks, consider adding an install step that sets this during installation once to the resolved python3 path. Do not set this to a relative path or `python3`. This will break when the virtualenv is activated as the pointed to python version is changed. In the virtualenv there is no virtualenvwrapper installed.

Reply:
Done